server/backend/api/app/crud/crud_access_control_privilege.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from api.app.models.model import FamAccessControlPrivilege, FamRole
+from sqlalchemy.orm import Session
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_delegated_admin_by_user_and_app_id(
+    db: Session, user_id: int, application_id: int
+) -> FamAccessControlPrivilege:
+    """
+    Find out from `app_fam.fam_access_control_privilege` if user is the delegated admin of the application.
+
+    :param user_id: primary id that is associated with the user.
+    :param application_id: primary id that is associated with the application.
+    :return: FamAccessControlPrivilege record, role information the user is able to manage
+    """
+    return (
+        db.query(FamAccessControlPrivilege)
+        .join(FamRole)
+        .filter(
+            FamAccessControlPrivilege.user_id == user_id,
+            FamRole.application_id == application_id,
+        )
+        .one_or_none()
+    )
+
+
+def get_delegated_admin_by_user_id(
+    db: Session, user_id: int
+) -> FamAccessControlPrivilege:
+    """
+    Find out from `app_fam.fam_access_control_privilege` if user is the delegated admin of any application.
+
+    :param user_id: primary id that is associated with the user.
+    :return: FamAccessControlPrivilege record, role information the user is able to manage
+    """
+    return (
+        db.query(FamAccessControlPrivilege)
+        .filter(
+            FamAccessControlPrivilege.user_id == user_id,
+        )
+        .all()
```

Comment:
Does this return a list of the FamAccessControlPrivileges? Method definition seems to say only one FamAccessControlPrivilege is returned.

Reply:
yeah, it should return a list, let me update it